Tests/PowerSkillTests/SplitImageTests/SplitImageTests.cs

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft. All rights reserved.  
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.  
+
+using AzureCognitiveSearch.PowerSkills.Common;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json.Linq;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace AzureCognitiveSearch.PowerSkills.Tests.SplitImageTests
+{
+    [TestClass]
+    public class SplitImageTests
+    {        
+        [TestMethod]
+        public async Task SplitBitmap()
```

Comment:
Use a `DataRowAttribute` to specify the image file names you use since they're the only thing changing between all those tests.

Reply:
I disagree, if you put only the file names, it's not that bad, you can concatenate the constant part of the path from the method body.